vars/restartPods.groovy

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env groovy
+
+class RestartPodsInput implements Serializable {
+    String projectName = ""
+    String targetApp = ""
+    String clusterUrl = ""
+    String clusterToken = ""
+}
+
+def call (Map input) {
+    call(new RestartPodsInput(input))
+}
+
+def call(RestartPodsInput input) {
+
+    assert input.targetApp?.trim() : "Param targetApp should be defined."
+    assert input.projectName?.trim() : "Param projectName should be defined."
+
+    openshift.withCluster(input.clusterUrl, input.clusterToken) {
+        openshift.withProject(input.projectName) {
+            openshift.selector("dc", input.targetApp).rollout().latest()
```

Comment:
hey @Gl4di4torRr, I think we need a different strategy for what it means to "restart" pods. Firstly, I think that, to make this more generic, we should target the pods in a ReplicationController rather than a DeploymentConfig. This ensures that we are always acting on the "active" version of the app. Secondly, using a `rollout` as a way to simply refresh the app could have some negative consequences in terms of possibly deploying a different version than what was deployed. I think instead, the proper way to execute a `restart` would be to simply do a rolling delete of each pod.